runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterTable/UsersRegisterTable.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { VFC } from 'react';
+
+import { Grid, Card } from '@mui/material';
+import { DataGrid, GridInputSelectionModel } from '@mui/x-data-grid';
+import { useSelector } from 'react-redux';
+
+import { usersSelector } from '#src-app/store/slices/Users';
+
+import { ROWS_PER_PAGE } from '../../UsersBrowseView/UsersBrowseView.utils';
+import { DataGridStyle} from './UsersRegisterTable.styles';
+import useUsersRegisterColumns from './useUsersRegisterColumns';
+
+interface TableProps {
+    page: number;
+    onPageChange: (page: number) => void;
+    pageSize: number;
+    onPageSizeChange: (pageSize: number) => void;
+    selections: GridInputSelectionModel;
+    handleSelectionChange: (selection: object) => void;
+    handleSelectedRolesChange: (id: number, value: string) => void;
+}
+
+const UsersRegisterTable: VFC<TableProps> = ({
```

Comment:
it's better to name interface the same as component (but with Props suffix ofc) - in case it's being exported during some refactor there could be name conflict